src/pages/settings/VatRate.tsx

```diff
@@ -1,43 +1,119 @@
-import { useRef } from 'react'
+import { useRef, useState } from 'react'
 import styled from 'styled-components'
 import { gql } from '@apollo/client'
 
 import { Typography, Button, Skeleton, Avatar, Icon } from '~/components/designSystem'
 import { useInternationalization } from '~/hooks/core/useInternationalization'
+import ErrorImage from '~/public/images/maneki/error.svg'
+import { GenericPlaceholder } from '~/components/GenericPlaceholder'
 import { theme, NAV_HEIGHT, HEADER_TABLE_HEIGHT } from '~/styles'
-import { useVatRateSettingQuery } from '~/generated/graphql'
+import {
+  EditOrganizationInvoiceTemplateDialogFragmentDoc,
+  useGetOrganizationInvoiceAndTaxInformationsQuery,
+} from '~/generated/graphql'
 import {
   EditOrganizationVatRateDialog,
   EditOrganizationVatRateDialogRef,
 } from '~/components/settings/EditOrganizationVatRateDialog'
+import {
+  EditOrganizationInvoiceTemplateDialog,
+  EditOrganizationInvoiceTemplateDialogRef,
+} from '~/components/settings/EditOrganizationInvoiceTemplateDialog'
+
+const MAX_FOOTER_LENGTH_DISPLAY_LIMIT = 200
 
 gql`
-  query vatRateSetting {
+  fragment OrganizationInvoiceTemplate on Organization {
+    invoiceFooter
+  }
+
+  query getOrganizationInvoiceAndTaxInformations {
     currentUser {
       id
       organizations {
         id
         vatRate
+        ...OrganizationInvoiceTemplate
+        ...EditOrganizationInvoiceTemplateDialog
       }
     }
   }
+
+  ${EditOrganizationInvoiceTemplateDialogFragmentDoc}
 `
 
 const VatRate = () => {
   const { translate } = useInternationalization()
-  const editDialogRef = useRef<EditOrganizationVatRateDialogRef>(null)
-  const { data, loading } = useVatRateSettingQuery()
-  const vatRate = data?.currentUser?.organizations
-    ? data?.currentUser?.organizations[0]?.vatRate
-    : 0
+  const [isTextTruncated, setIsTesxtTruncated] = useState(true)
+  const editVATDialogRef = useRef<EditOrganizationVatRateDialogRef>(null)
+  const editInvoiceTemplateDialogRef = useRef<EditOrganizationInvoiceTemplateDialogRef>(null)
+  const { data, error, loading } = useGetOrganizationInvoiceAndTaxInformationsQuery()
+  const organization = (data?.currentUser?.organizations || [])[0]
+  const vatRate = organization?.vatRate || 0
+  const invoiceFooter = organization?.invoiceFooter || ''
+
+  const renderInvoiceFooter = () => {
```

Comment:
I wonder if this couldn't be a component ?

Reply:
Looks like something we could reuse

Reply:
on it!

Reply:
Done [in the last commit](https://github.com/getlago/lago-front/pull/209/commits/61f4b4132cee689a53abc7abd3cd84d4f2b831ab), let me know what you think!